### ISSUE TITLE
error with prettier 2.0.1

### ISSUE THREAD

**Issue by kapilpipaliya** [sveltejs/prettier-plugin-svelte#84]
I get errors when trying with prettier
`src/App.svelte: Error: unknown node type: MemberExpression `

**Comment by eps1lon** [comment]
Using `prettier@2.0.1` I get:
```bash
{
    "type": "Literal",
    "start": 362,
    "end": 366,
    "value": true,
    "raw": "true",
    "isJS": true
}
Error: unknown node type: Literal
```
when formatting
```svelte
<script>
	import Drawer from 'redacted';
	import Nav from './Nav.svelte';

	export let open = false;
	export let segment;
</script>

<style>
	:global(.drawer) :global(nav) {
		height: 100%;
	}
</style>

<Drawer bind:open>
	<Nav {segment} initalFocus={true} on:itemClick={() => (open = false)} />
</Drawer>

```

**Comment by b2m9** [comment]
Same here.

Workaround for VSCode: Install an older version of Pretier in your project. The VSCode Prettier plugin will use the version specified in your `package.json` instead of the latest version that comes with the plugin.

**Comment by jmcudd** [comment]
What version of Prettier? I've tried older versions and it still doesn't work.

**Comment by raphaelschaad** [comment]
Is an official maintainer of prettier-plugin-svelte looking into making it compatible with Prettier 2?

@jmcudd 1.19.1 worked for me.

**Comment by justinmoon** [comment]
@raphaelschaad Perhaps this is something @Conduitry plans to add when making it an "official plugin" #86

**Comment by QuantumInformation** [comment]
I'm having no luck with prettier for even simple files like

```
<script>
    import { link } from "svelte-routing"

    //since this is in the authenticated HOC $authUserStore is guaranteed to be there

    export let user


</script>

<div style="border: 1px solid black;padding: 1em;margin-bottom: 1em">
    User:
    <a use:link href="{`profile?username=${user.username}`}">
        <span>{user.displayName || user.uid}</span>
    </a>
</div>
```

using 
    "prettier": "^1.19.1",
    "prettier-plugin-svelte": "^0.7.0",

**Comment by Conduitry** [comment]
Fixed in 1.0.0 🎉

**Comment by QuantumInformation** [comment]
when running 1.0.0 it with

{#await promise}
    <DefaultSpinner />
{:then user}
    <h1>{user.displayName}</h1>
{:catch error}
    <p style="color: red">{error.message}</p>
{/await}


I get this output


{#await promise}
    <DefaultSpinner />
{:then [object Object]}
    <h1>{user.displayName}</h1>
{:catch [object Object]}
    <p style="color: red">{error.message}</p>
{/await}

**Comment by Conduitry** [comment]
This issue is for the plugin not running at all in Prettier 2, with `unknown node type` exceptions. If you've found another bug that doesn't already have an issue, please open a new one.

**Comment by Conduitry** [comment]
FWIW it sounds like that is related to what is attempting to be solved in #83.

**Comment by QuantumInformation** [comment]
oh nice thx, though mine was unique
